src/bp/core/database/helpers.ts

```diff
@@ -111,6 +119,9 @@ export const patchKnex = (knex: Knex): KnexExtended => {
   }
 
   const date: Knex.Date = {
+    set: (date?: Date) => (date ? date.toISOString() : undefined),
```

Comment:
I created `date.set` and `date.get` so we can use it the same way as `json.set` and `json.get`.  It seems like a good idea to have at least those two functions for every "complex" types. Otherwise you end up writing code a certain way to handle the json, and another way to handly the dates.